python-package/lightgbm/dask.py

```diff
@@ -593,14 +755,23 @@ def fit(
         X: _DaskMatrixLike,
         y: _DaskCollection,
         sample_weight: Optional[_DaskCollection] = None,
+        init_score: Optional[_DaskCollection] = None,
+        eval_set: Optional[List[Tuple[_DaskCollection, _DaskCollection]]] = None,
```

Comment:
Could you please also pass through `eval_names`? Ideally I'd like these methods to have the same argument position as `lgb.sklearn.LGBMModel.fit()`.

https://github.com/microsoft/LightGBM/blob/fd094d2719a04b984b376a665b6c3e5e1aff2967/python-package/lightgbm/sklearn.py#L544

and can you please do this in `DaskLGBMRegressor.fit()` and `DaskLGBMRanker.fit()` as well?

Reply:
Addressed in 6737acd5